packages/kit/src/module/define.ts

```diff
@@ -67,7 +67,25 @@ export function defineNuxtModule<OptionsT extends ModuleOptions> (definition: Mo
     }
 
     // Call setup
-    await definition.setup?.call(null as any, _options, nuxt)
+    const setupStart = process.hrtime()
+    const res = await definition.setup?.call(null as any, _options, nuxt) ?? {}
+    const setupEnd = process.hrtime(setupStart)
+
+    // Measure setup time
+    const setupTime = Math.round((setupEnd[0] * 1000000000 + setupEnd[1]) / 10000) / 100 /* round by two digits */
+    if (setupTime > 1000) {
+      console.warn(`Module \`${uniqueKey || '<no name>'}\` took \`${setupTime}ms\` to setup!`)
```

Comment:
```suggestion
      logger.warn(`Module \`${uniqueKey || '<no name>'}\` took \`${setupTime}ms\` to setup!`)
```

Reply:
Why not using our logger directly?

Reply:
They are actually same channels since we mock.